docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -81,6 +83,11 @@ Property Name                                      Description
                                                    By default, the value corresponds to the default protocol version
                                                    used in the underlying Cassandra java driver.
                                                    Possible values include ``V3``, ``V4``, ``V5``, ``V6``.
+
+``cassandra.load-policy.use-dc-aware``             Set to ``true`` to use ``DCAwareRoundRobinPolicy``, defaults
+                                                   to ``false``.
+
+``cassandra.load-policy.dc-aware.local-dc``        The name of the local datacenter for ``DCAwareRoundRobinPolicy``.
```

Comment:
also .. this is only required if use-dc-aware is set to true?

Reply:
It becomes required when `cassandra.contact-points` is set (=always required now).

Reply:
So we should say that it is required explicitly.